pkg/cluster/nodes/node.go

```diff
@@ -88,6 +88,11 @@ func (n *Node) CopyTo(source, dest string) error {
 	return docker.CopyTo(source, n.nameOrID, dest)
 }
 
+// CopyFrom copies the source file on the node to dest on the host
```

Comment:
note that this does have limitations around symlinks, I initially implemented this method and then dropped it for log dumping because of this. (we should consider a similar implementation here in the future, but this seems reasonable for now)

Reply:
Added a note to keep track of this.
We should consider that this should probably go away when kubeadm automatic copy certs lands, otherwise it should be refactored in something more robust in the long term